src/dongyeon-lee/230223/P138476.kt

```diff
@@ -0,0 +1,10 @@
+package `dongyeon-lee`.`230223`
+
+/* 할인 행사 */
+class Solution {
+    fun solution(want: Array<String>, number: IntArray, discount: Array<String>) =
+        discount.toList().windowed(10).count { window ->
+            val count = window.groupingBy { it }.eachCount()
+            want.filterIndexed { index, s -> count[s] == number[index] }.size == want.size
```

Comment:
```suggestion
            want.withIndex().count { (idx, value) -> count[value] == number[idx] } == want.size
```

이렇게 하면 filterIndexed {...} 로 나오는 List 객체를 안만들 수 있기는 한데..
뭐 별 차이는 없을듯요. 동연님 지금 코드도 ㄱㅊ